packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 # Changelog
+## 3.1.1 - Dec 27, 2021
+This is a hotfix version.
+
+The Azure Bot service is upgraded and does not support some old NodeJs versions in some regions any more. This hotfix solves the problem that bot service is not working in those regions which does not support old NodeJs version.
```

Comment:
Actually it is not Azure Bot service, but Azure App Service instead.

Reply:
thank you, fixed.

Reply:
change all "old" -> "older"

Reply:
ok, fixed.